Exiled.API/Features/Room.cs

```diff
@@ -76,7 +76,7 @@ public class Room : MonoBehaviour
         /// <summary>
         /// Gets a <see cref="IEnumerable{T}"/> of <see cref="Player"/> in the <see cref="Room"/>.
         /// </summary>
-        public IEnumerable<Player> Players => Player.List.Where(player => player.IsAlive && player.CurrentRoom.Transform == Transform);
+        public IEnumerable<Player> Players => Player.List.Where(player => player.IsAlive && !(player.CurrentRoom is null) && player.CurrentRoom.Transform == Transform);
```

Comment:
Pattern matching sucks in C# 7.2 or whatever is it, just `!= null`

Reply:
or !(is null)

Reply:
`!= null` is just better in this case, cause it doesn't destroys the pattern matching

Reply:
Well, I'm bumping the project to C# 9 and re-working all of the null checks to use pattern matching `is not null` or `is null` anyways so, it won't really matter, I'd just be changing this either way.